ansible/roles/ros_install/tasks/main.yml

```diff
@@ -53,3 +53,6 @@
 
 - name: rosdep init
   command: rosdep init creates=/etc/ros/rosdep/sources.list.d/20-default.list
+  
+- name: Update dependencies
+  shell: bash -c "rosdep update -y"
```

Comment:
@AndriyPt just a question for you - why do you use `shell: bash -c "your command"` instead of just `command: ` as we did with @markpitchless